ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -590,6 +600,10 @@ qualifiedName
     : ident (DOT ident)*                             #identsAsQualifiedName
     ;
 
+clusterQualifiedName
```

Comment:
the other to do this would be to allow the use of : in index names.

Reply:
I've considered this option. The reason why I don't think it work very well is because we really only want : in search commands. Changing the symbols allow-list for index names would also change the name we allow for user to create a new index. We'll need to reject the : symbol else where.

Which do you think is better?

Reply:
allowed use of : in index name